### DISCUSSION TITLE
Default syntax highlighter

### DISCUSSION THREAD

**Discussion by silverqx** [facebook/docusaurus#5122] [Q&A] [answered]
It is possible to define the default syntax highlighter?

I want to achieve this, when I define code block by 3 backticks without language meta string or code block intended with tab or 4 spaces I want to have this code block highlighted by my defined syntax highlighter eg `cpp`.

Currently, these blocks are unhighlighted.

If it is not possible with some simple configuration I also welcome some workaround.

Thank you for any advice or suggestions 🙏

**Comment by Josh-Cena** [comment, marked as answer]
https://docusaurus.io/docs/next/api/themes/configuration#default-language

Do check the docs and utilize the search function 😂

**Reply by silverqx** [reply]
I swear God that I searched 20mins in docs, github issues, discussions, I also tried `defaultLang` 👀 and it is described right away in the docs 😁 I'm still trying to orient in Docusaurus 🕺 @Josh-Cena Thank you, it works great. 🙏

**Reply by Josh-Cena** [reply]
Lol I agree that overall the `api` section of the documentation looks like a misty sea with so much information...

**Reply by silverqx** [reply]
@Josh-Cena It is not so bad, one has to little dig into and after few hours you get it in the blood, I have discovered Docusaurus yesterday and decided to use it before Jekyll, I like it a lot 😎

**Reply by slorber** [reply]
Thanks @Josh-Cena 

We have exhaustive api in the api section but not all features are available in the regular docs/guides.

If you feel we should add this in the guides, please send a doc PR ;)
